pkg/capabilities/consensus/ocr3/datafeeds/feeds_aggregator.go

```diff
@@ -140,6 +145,68 @@ func (a *dataFeedsAggregator) Aggregate(previousOutcome *types.AggregationOutcom
 	}, nil
 }
 
+func (a *dataFeedsAggregator) extractSignersAndPayloads(observations map[ocrcommon.OracleID][]values.Value, fConsensus int) ([][]byte, int, map[ocrcommon.OracleID]values.Value) {
+	payloads := make(map[ocrcommon.OracleID]values.Value)
+	signers := make(map[[addrLen]byte]int)
+	mins := make(map[int]int)
+	for nodeID, nodeObservations := range observations {
+		// we only expect a single observation per node - a Streams trigger event
+		if len(nodeObservations) == 0 || nodeObservations[0] == nil {
+			a.lggr.Warnf("node %d contributed with empty observations", nodeID)
+			continue
+		}
+		if len(nodeObservations) > 1 {
+			a.lggr.Warnf("node %d contributed with more than one observation", nodeID)
```

Comment:
should there be a continue here? if not, why not (ie add a comment)

Reply:
Technically not because we can just ignore extra values. But maybe better to actually treat that value as bad (misconfigured workflow spec?) - added continue.